src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -1,39 +1,29 @@
 @echo off
 setlocal
 
-set PROJECT_DIR=%~1
-set DOTNET_CMD=%~2
-set TOOLRUNTIME_DIR=%~3
-IF [%BUILDTOOLS_TARGET_RUNTIME%]==[] set BUILDTOOLS_TARGET_RUNTIME=win7-x64
+set DOTNET_CMD=%~1
+set TOOLRUNTIME_DIR=%~2
 set BUILDTOOLS_PACKAGE_DIR=%~dp0
-set PORTABLETARGETS_VERSION=0.1.1-dev
 set ROSLYNCOMPILERS_VERSION=1.2.0-beta1-20160202-02
```

Comment:
mm we don't really have one really other than monitoring PR's and trying to make sure that both the Net Core compilers and the full framework ones stay in sync. Do you have any specific idea in mind?